Makefile

```diff
@@ -19,7 +19,7 @@ BUILDTAGS += $(EXTRA_BUILDTAGS)
 
 COMMIT ?= $(shell git describe --dirty --long --always)
 VERSION ?= $(shell cat ./VERSION)
-LDFLAGS_COMMON := -X main.gitCommit=$(COMMIT) -X main.version=$(VERSION)
+LDFLAGS_COMMON := -X main.gitCommit=$(COMMIT)
```

Comment:
With this change will we still need the `VERSION` variable in Makefile. ?

Another concern  is,  the patch will make changes in https://github.com/opencontainers/runc/commit/9d9273c926450a2f6f658768e3238b7082ec878d ineffective. The use case was someone should be able to pass a custom version without editing the VERSION file.

Reply:
@thaJeztah it looks like `VERSION` should be removed from the Makefile altogether.

Also, this needs another rebase.